docs/source/contributing.rst

```diff
@@ -0,0 +1,44 @@
+Contribute to QuantEcon.py
+==========================
+
+If you would like to contribute to :ref:`QuantEcon.py<https://github.com/QuantEcon/QuantEcon.py>`, 
+a good place to start is the :ref:`project issue tracker<http://github.com/QuantEcon/QuantEcon.py/issues>`. 
+
+Set up a Conda development environment
+--------------------------------------
+
+One of the advantages of the :ref:`Anaconda Python environment<https://www.anaconda.com/distribution/>` is that it is
+cheap to set up (and discard) Python environments for development versions of packages and populate them with your
+favorite scientific tools. For example, if you're working on QuantEcon.py you might find it useful to set up an
+environment (containing NumPy, SciPy, etc.) that uses your development version rather than the default ones. 
+
+This facilitates contributing to QuantEcon.py without worrying about corrupting the Python environment on which your other work depends.
+
+Write tests
+-----------
+
+All functions and methods contributed to QuantEcon.py should be paired with tests to verify that they are functioning correctly.
+
+Write documentation
+-------------------
+
+We try to maintain a simple and consistent format for inline documentation, known in the Python world as docstrings. 
+The format we use is known as :ref:`numpydoc<https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt>`. 
```

Comment:
This page no longer exists.

Reply:
thanks @oyamad migrating content at the moment from the QuantEcon org site and will proof read. Appreciate the comment.